comms/ModFileAttribute.cpp

```diff
@@ -70,6 +70,11 @@ class CModFileAttribute : public IAttribute, public IAttributeHistory, public cl
     {
     }
 
+    int PreProcess(PREPROCESS_TYPE action, const TCHAR * overrideEcl, IAttributeVector & attrs, IAttributeBookkeep & attrProcessed, Dali::CEclExceptionVector & errs) const
+    {
+        return 0;
+    }
+
```

Comment:
This file already has a "PreProcess" method whose signature needs to change to match this one.